docs/docs/Getting-Started.md

```diff
@@ -0,0 +1,36 @@
+---
+id: Getting-Started
+title: Getting Started
+---
+Hello Newcomer, here are some tips to help you get started contributing to this project.
+
+## Get to know the project
+
+Here's what you can do to know more about the project:
+- Read Documentation available on [GitHub Wiki](https://github.com/systers/mentorship-backend/wiki) and [README](https://github.com/systers/mentorship-backend/blob/develop/README.md);
+- You can join the [#mentorship-system](https://systers-opensource.slack.com/messages/CAE8QK41L/) channel on [Systers Open Source Slack](http://systers.io/slack-systers-opensource/), to see or participate in the project's discussion;
+- You can browse the code on GitHub or even on your workspace after [cloning it](https://github.com/systers/mentorship-backend/wiki/Fork,-Clone-&-Remote#clone).
+
+## Choose a working item
+
+1. Check the [available issues](https://github.com/systers/mentorship-backend/issues?q=is%3Aissue+is%3Aopen+label%3A%22Status%3A+Available%22) (that anyone can contribute to) or [first timers only issues](https://github.com/systers/mentorship-backend/labels/First%20Timers%20Only) (just for first time contributors in this project);
+2. Choose one issue you want to work on;
+3. Ask maintainers, on the issue's comment section, if you can work on it;
+4. Once you get approval you can start working on it!
+
+## Start working
+
+Before you start working check the [Contribution Guidelines](https://github.com/systers/mentorship-backend/blob/develop/.github/CONTRIBUTING.md) to make sure you can follow the best practises.
+
+**In short:**
+
+1. [Fork the project](https://github.com/systers/mentorship-backend/wiki/Fork,-Clone-&-Remote#fork) into your profile;
+2. [Clone the project](https://github.com/systers/mentorship-backend/wiki/Fork,-Clone-&-Remote#clone) into your workspace on your computer;
+3. [Setup remotes](https://github.com/systers/mentorship-backend/wiki/Fork,-Clone-&-Remote#remote) to keep your `develop` branch with systers repository;
+4. Create a specific branch based from `develop` branch for your specific feature;
+5. Start coding;
+6. Make sure you follow this [Commit Message Style Guide](https://github.com/systers/mentorship-backend/wiki/Commit-Message-Style-Guide);
```

Comment:
change the link of `Commit Message Style Guide` to https://mentorship-backend.surge.sh/Commit-Message-Style-Guide